src/Core/Services/Implementations/LaunchDarklyFeatureService.cs

```diff
@@ -214,9 +228,21 @@ void SetCommonContextAttributes(ContextBuilder builder)
                         builder.Add(ldServiceAccount.Build());
                     }
 
-                    if (_currentContext.OrganizationId.HasValue)
+                    if (overridingOrganizationId.HasValue || _currentContext.OrganizationId.HasValue)
                     {
-                        var ldOrg = LaunchDarkly.Sdk.Context.Builder(_currentContext.OrganizationId.Value.ToString());
+                        ContextBuilder ldOrg;
+                        if (overridingOrganizationId.HasValue)
```

Comment:
💭 Same question -- should the override be respected here? There's an organization ID tied to the service account.